### PR TITLE
Apache artemis origin header needed

### PR DESCRIPTION
**Pull request by bonky42**
Encounter a problem while using apache artemis (last version), it complains that the header Origin is null. It seems now, unless you remove cors protection there is a need to add this header.
Content of the patch:

- Adding the origin header
- making it configurable in the metadata

Signed-off-by: Bernard Chesnoy <bernard.chesnoy@gmail.com>

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -23,6 +23,7 @@
 - Add Azure Pipelines Scaler ([#1706](https://github.com/kedacore/keda/pull/1706))
 - Add OpenStack Metrics Scaler ([#1382](https://github.com/kedacore/keda/issues/1382))
 - Added basic, tls and bearer authentication support to the Prometheus scaler [#1559](https://github.com/kedacore/keda/issues/1559)
+- Add header Origin to apache artemis scaler
```
Comment (zroubalik) [inline CHANGELOG.md:26]:
```suggestion
- Add header Origin to Apache Artemis scaler [#1796](https://github.com/kedacore/keda/pull/1796)
```

**Review by zroubalik — APPROVED** [review]
Looking good, thanks @bonky42 

I am just curious, is this something that is needed for newer versions of Artemis and it wasn't needed before? If that's the case, does the code introduced in this PR work with the older versions of Artemis?

File: CHANGELOG.md (hunk 1 above)
Comment (bonky42) [reply CHANGELOG.md]: https://issues.apache.org/jira/browse/ARTEMIS-2779
I find it in this issue, I'll try with 2.11 and resolve the conversation.

Comment (bonky42) [reply CHANGELOG.md]: tested on 2.11.0, it works

**Pull request merged**